### ISSUE TITLE
Instrument buffer is filled with junk if a scan is stopped prematurely

### ISSUE THREAD

**Issue by stevenaleung** [Orchard-Ultrasound-Innovation/IntensityScans.jl#28]
Options are:
1. Flushing the buffer before data acquisition begins
2. Flushing the buffer after an error

For option 1, the buffer bug may still present a problem if a non-acquisition function is called after a premature scan stop

**Comment by mofii** [comment]
@stevenaleung Is this the receive buffer of the oscilloscope?

**Comment by stevenaleung** [comment]
Yes. Maybe we can close this issue in favor of Orchard-Ultrasound-Innovation/TcpInstruments.jl#139
